warnings.rb

```diff
@@ -1,9 +1,9 @@
 
-a = { "hello" => "world" }
+a={ "hello" => "world" }
```

Comment:
_[farcy v0.1b](https://github.com/appfolio/farcy)_

* Lint/UselessAssignment: Useless assignment to variable - `a`.
* Style/SpaceAroundOperators: Surrounding space missing for operator '='.
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.